pkg/cmd/flags/policy.go

```diff
@@ -0,0 +1,291 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/pkg/events"
+)
+
+// PolicyFile is the structure of the policy file
+type PolicyFile struct {
+	Name          string   `yaml:"name"`
+	Description   string   `yaml:"description"`
+	Scope         []string `yaml:"scope"`
+	DefaultAction string   `yaml:"defaultAction"`
+	Rules         []Rule   `yaml:"rules"`
+}
+
+// Rule is the structure of the rule in the policy file
+type Rule struct {
+	Event  string   `yaml:"event"`
+	Filter []string `yaml:"filter"`
+	Action string   `yaml:"action"`
+}
+
+// PrepareFilterMapForPolicies prepares the FilterMap for the policies
+func PrepareFilterMapFromPolicies(policies []PolicyFile) (FilterMap, error) {
+	filterMap := make(FilterMap)
+
+	if len(policies) == 0 {
+		return nil, errfmt.Errorf("no policies provided")
+	}
+
+	if len(policies) > 64 {
+		return nil, errfmt.Errorf("too many policies provided, there is a limit of 64 policies")
+	}
+
+	for i, p := range policies {
+		err := validatePolicy(p)
```

Comment:
I think we are missing validation that the same policy name is not used more than once